openscenario/openscenario_interpreter/src/syntax/storyboard.cpp

```diff
@@ -44,8 +45,13 @@ Storyboard::Storyboard(const pugi::xml_node & node, Scope & scope)
 
 auto Storyboard::run() -> void
 {
-  for (auto && story : elements) {
-    story.evaluate();
+  for (size_t index{0}; auto && story : elements) {
```

Comment:
This repository is compiled with C++17. Don't take advantage of C++20 features yet.
```suggestion
  std::size_t index = 0;
  for (auto && story : elements) {
```